test/tools/fixtures/process_resource_script.in.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+/* eslint-disable no-undef */
+/* eslint-disable no-unused-vars */
+const driverPath = DRIVER_SOURCE_PATH;
+const func = FUNCTION_STRING;
+const name = NAME_STRING;
+const uri = URI_STRING;
+
+const { MongoClient } = require(driverPath);
+const process = require('node:process');
+const v8 = require('node:v8');
+const util = require('node:util');
+const timers = require('node:timers');
+const fs = require('node:fs');
+const chai = require('chai');
```

Comment:
```suggestion
const { expect } = require('chai');
```

I think it would be nice to not have to prefix expect with `chai.`

Reply:
switched it to be an argument!

Reply:
I think Neal was saying that we can use `const { expect } = require('chai');` here.  Then each test function will work because when it is executed, `expect` will be defined.  No need to pass `chai` as an argument